recipe/meta.yaml

```diff
@@ -25,6 +27,11 @@ requirements:
   run:
     - python
 
+test:
+  commands:
+    - dir %PREFIX%\pythoncom%CONDA_PY%.dll
```

Comment:
Correct `if exist` and `if not exist` are special syntax on Windows for `if` in particular. `exist` is not a command by itself on Windows.

ref: http://ss64.com/nt/if.html